test/ntp_client_test.rb

```diff
@@ -421,41 +447,150 @@
       expect(subject.TestNtpServer("server", "")).to eql(false)
     end
 
-    context "when given verbosity is no ui" do
-      it "doesn't show any dialog" do
-        expect(Yast::Popup).to receive(:Feedback).never
-      end
+    it "doesn't show any dialog if given verbosity is :no_ui" do
```

Comment:
no, just the no_ui case to return directly reachable_ntp_server?, but i can i can cover this cases also.